src/xinterpreter.cpp

```diff
@@ -357,7 +361,28 @@ __get_cxx_version ()
 
     void interpreter::init_includes()
     {
+        // Add the standard include path
         Cpp::AddIncludePath((xeus::prefix_path() + "/include/").c_str());
+
+        // Get include paths from environment variable
+        const char* non_standard_paths = std::getenv("XEUS_SEARCH_PATH");
+        if (!non_standard_paths) {
+            non_standard_paths = "";
+        }
+
+        if (std::strlen(non_standard_paths) > 0)
+        {
+            // Split the paths by colon ':' and add each one
+            std::istringstream stream(non_standard_paths);
+            std::string path;
+            while (std::getline(stream, path, ':'))
+            {
+                if (!path.empty())
+                {
+                    Cpp::AddIncludePath(path.c_str());
+                }
+            }
+        }
```

Comment:
We can simplify this by a lot, reduce the mount braces and also consider non-unix path delimiters.

Reply:
Done